jsapp/js/account/stripe.api.ts

```diff
@@ -50,9 +54,13 @@ export async function getOrganization() {
 /**
  * Start a checkout session for the given price and organization. Response contains the checkout URL.
  */
-export async function postCheckout(priceId: string, organizationId: string) {
+export async function postCheckout(
+  priceId: string,
+  organizationId: string,
+  quantity = 1
+) {
   return fetchPost<Checkout>(
-    `${endpoints.CHECKOUT_URL}?price_id=${priceId}&organization_id=${organizationId}`,
+    `${endpoints.CHECKOUT_URL}?price_id=${priceId}&organization_id=${organizationId}&quantity=${quantity}`,
```

Comment:
Part of me wants this to be a key-value mapping to represent the query params. It doesn't bother me when it's only 1 or 2. Feel free to ignore.

```
interface checkoutParams {
     priceId: string,
     etc
     
`${endpoints.CHECKOUT_URL}?new URLSearchParams(params)`
```